lib/puppet/functions/extlib/dump_params.rb

```diff
@@ -0,0 +1,18 @@
+# @summary this function is used to get a list of parameters passed to a class.  This allows one to easily
+#   transofrm class parameteres into a jsone/yaml config file
```

Comment:
```suggestion
#   transofrm class parameteres into a json/yaml config file
```

Reply:
out of curiosity, could you write a sentence or two and explain the intention of that function?

Reply:
i have added a bit more text to the summery including some examples, let me know if its to verbose or still unclear